assets/js/components/App.tsx

```diff
@@ -30,6 +33,8 @@ class AppRoutes extends React.Component {
             <Route path="dashboard" element={<PlacesPage />}></Route>
             <Route path="alerts" element={<AlertsPage />}></Route>
             <Route path="alerts/:id" element={<AlertDetails />}></Route>
+            <Route path="pending" element={<PendingScreensPage />}></Route>
```

Comment:
Added a new component with just a header for the pending page. I believe the configure screens page can be inserted with [this PR](https://github.com/mbta/screenplay/pull/235/files)